src/app/courses/view-courses/courses-view.component.ts

```diff
@@ -44,11 +47,14 @@ export class CoursesViewComponent implements OnInit, OnDestroy {
       });
       this.progress = progress;
       this.isUserEnrolled = this.checkMyCourses(course._id);
+      this.canManage = this.currentUser.isUserAdmin;
```

Comment:
@svlesiv Here it is checking the user is admin or not. `addedBy` is the username of creator of resources.